### DISCUSSION TITLE
Anti-aliasing

### DISCUSSION THREAD

**Discussion by SwiftsNamesake** [ad-si/Brillo#21] [Ideas]
Hey! Thanks for this library!

Would it be possible to add anti-aliasing support?

**Comment by SwiftsNamesake** [comment]
Just realised you've already addressed this [here](https://www.reddit.com/r/haskell/comments/1h23ti9/brillo_painless_2d_graphics_fork_of_gloss/)

**Comment by ad-si** [comment]
Here is the corresponding issue: https://github.com/ad-si/Brillo/issues/2
